Mathlib/RingTheory/Nilpotent/Exp.lean

```diff
@@ -46,8 +50,8 @@ It provides meaningful (non-junk) values for nilpotent elements. -/
 noncomputable def exp (a : A) : A :=
   ∑ i ∈ range (nilpotencyClass a), (i.factorial : ℚ)⁻¹ • (a ^ i)
 
-theorem exp_eq_truncated {a : A} {k : ℕ} (h : a ^ k = 0) :
```

Comment:
If you've chosen not to deprecate this, please explain why in the PR description